### ISSUE TITLE
Date/Time

### ISSUE THREAD

**Issue by jaclynsaunders** [BCODMO/bcodmont#9]
Reminder: Date/Time still do not exist in the ontology. These need to be handled.

**Comment by kaiiam** [comment]
I presume we should do this in the operational module? Do we want all possible variants:

`datetime`, `date`, `time`, `year`, `month`, `day`?

**Comment by ashepherd** [comment]
i think we do have some others too like yearday, etc. Let me look at our master parameters to find all those. i don't think we'd have anything that ins't in XSD (fingers crossed)

**Comment by ashepherd** [comment]
I started here: https://www.bco-dmo.org/search/parameter/date?size=20&sort=relevance

these are the funky ones: 

monthYear https://www.bco-dmo.org/parameter/1083
julian day (year 0) https://www.bco-dmo.org/parameter/1808
julian day https://www.bco-dmo.org/parameter/866
date range https://www.bco-dmo.org/parameter/542026
year day https://www.bco-dmo.org/parameter/1116
day length https://www.bco-dmo.org/parameter/871
year day as decimal year https://www.bco-dmo.org/parameter/1930
duration https://www.bco-dmo.org/parameter/555754

- there are some variants on these above in the list for UTC or GMT
- then the standard year, month, day etc like you said

**Comment by kaiiam** [comment]
Thanks @ashepherd I appreciate it. Are we good for now to just add new BSM terms for these? Or do we want to have this integrate into your other workflows to enforce the use of frictionless data types etc. It might not be necessary now but we should think about having that connection at some point if useful.

**Comment by ashepherd** [comment]
yeah definitely lets' talk through this. I can show you how the Frictionlessdata types are getting used in the metadata now
